asreview/webapp/api.py

```diff
@@ -713,35 +713,103 @@ def api_random_prior_papers(project):  # noqa: F401
     the already labeled items.
     """
 
+    subset = request.args.get("subset", default=False, type=bool)
```

Comment:
Not clear why this is of type `bool`. I would expect the subset to be similar to history subsets for example.

Reply:
This is an indicator of project mode. In the simulation and exploration modes, the returned random records need to be extracted from relevant and irrelevant subsets.